### DISCUSSION TITLE
Jwarmup

### DISCUSSION THREAD

**Discussion by hwominitrix** [alibaba/dragonwell11#681] [Q&A]
When JWarmup in Dragonwell8 will be get in Dragonwell11?

**Comment by LeymerNiu** [comment]
jwarmup在dragonwell11中不支持了吗？

**Comment by hwominitrix** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
